packages/docusaurus/src/client/exports/__tests__/isInternalUrl.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import isInternalUrl from '../isInternalUrl';
+
+describe('isInternalUrl', () => {
+  test('should be true for root relative links', () => {
+    expect(isInternalUrl('/foo/bar')).toBeTruthy();
```

Comment:
I think this might be incorrect in the cases it is a relative link from the root of the domain. For example your site is a www.example.com/website/ and your path is /hello/foo/ which is intended to lead to www.example.com/hello/foo. Maybe we need to compare with the baseUrl to decide if it's an internal link? WDYT?

Reply:
I thought about it, but how do we get the base url in this util function? Pass baseUrl as an argument? Hmm, I don’t know, it seems to me then users need to only use relative links for this.

Reply:
We can read the `baseUrl` from the config, so users don't have to pass it in. Maybe check if the string values starts with `baseUrl`? Not sure too.

Reply:
I would prefer to leave it as it is, if in the future users encounter such a case, then we will try to solve it.

Reply:
Sounds good